### PR TITLE
Add Kubernetes deployment files

### PR DESCRIPTION
**Pull request by vfreex**
Also prints logs to stderr,
and doesn't create server.pid on start.

### CODE REVIEW

File: server/src/tools/daeml.cpp [hunk 1]
```diff
@@ -12,11 +12,11 @@
 void closeall(int fd)
```
Comment (shiywang) [inline server/src/tools/daeml.cpp]: I dont think this `daeml` is necessary as we are switching to use `docker-compose` or `kubernetes`

**Comment by shiywang** [comment]
Is this still WIP? or we are ready to go?

File: server/src/tools/daeml.cpp (hunk 1 above)
Comment (vfreex) [reply server/src/tools/daeml.cpp]: I accidentally committed this file.
Agreed. With Docker/Kubernetes/systemd, home-brewed process supervisor is not needed.

**Comment by vfreex** [comment]
> Is this still WIP? or we are ready to go?

It is not fully featured (no liveness/readiness check, no monitoring, no startup order control, no auto DB schema import, ...), but it works and I tried it on my home server.

I would prefer to merge it first then improve it later.

**Comment by vfreex** [comment]
Merged based on @shiywang 's feedback.

**Pull request merged**